.circleci/config.yml

```diff
@@ -0,0 +1,161 @@
+# CircleCI 2.0 configuration file. See <https://circleci.com/docs/2.0/language-python/>.
+version: 2
+jobs:
+  build_python2:
+    parallelism: 3
+    docker:
+      - image: python:2.7.14
+
+    steps:
+      - checkout
+
+      - restore_cache:
+          key: v1-py2-{{ checksum "setup.py" }}
+
+      - run:
+          name: Create a virtualenv
+          command: |
+            mkdir -p /tmp/venv/openfisca_france
+            virtualenv /tmp/venv/openfisca_france
```

Comment:
Why create a virtualenv ? In my mental model, OpenFisca-France will be the only thing running in this Docker Instance. I'm curious of why we are going through the virtualenv steps ...